<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="waypoint_file_line_edit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="waypoint_file_label">
      <property name="text">
       <string>Waypoint File(.json)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="enable_waypoint_collection_checkbox">
      <property name="text">
       <string>Enable Waypoint Collection</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="save_waypoint_file_btn">
      <property name="text">
       <string>Set Waypoint File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="waypoint_info_text_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>421</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="waypoint_info_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Waypoint Information</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
